src/components/ProjectList/listOfProjects.js

```diff
@@ -689,6 +689,13 @@ const projectList = [
     projectLink: 'https://github.com/zero-to-mastery/CSS-Art#instructions',
     description: 'General Edition - A CSS art challenge, for all skill levels.',
     tags: ['CSS', 'HTML', 'Javascript', 'Beginner'],
+  },
+  {
+    name: 'Chat-e2ee',
+    imageSrc: 'https://repository-images.githubusercontent.com/271544524/44353a80-d451-11ea-815c-594e4a0c5fb8',
+    projectLink: 'https://github.com/muke1908/chat-e2ee',
```

Comment:
Hey @muke1908 Can you please point the project link to either <project_link>/contribute or <project_link_issues><tag=good first or tag=help wanted>
That will be more consistent with how we add links for the projects.

Reply:
Github `/contribue` section doesn't show good first issue for some reason, So I have added issue links that are tagged is good first issue.